.github/workflows/build-and-deploy-new-acct.yml

```diff
@@ -0,0 +1,74 @@
+# temporary workflow to start deploying to the new prod account.
+name: Build and deploy new account
+on:
+  push:
+    branches:
+      - master
+jobs:
+  buildSite:
+    env:
+      GOPATH: ${{ github.workspace }}/go
+    name: Install deps and build site
+    runs-on: buildjet-4vcpu-ubuntu-2004
+    steps:
+
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '16.x'
+
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.19.x
+
+      - uses: peaceiris/actions-hugo@v2
+        with:
+          hugo-version: '0.96.0'
+          extended: true
+
+      - name: Configure Git
+        run: |
+          git config --global url."https://${{ secrets.PULUMI_BOT_TOKEN }}:x-oauth-basic@github.com/pulumi/pulumi-hugo-internal".insteadOf "https://github.com/pulumi/pulumi-hugo-internal"
+      - name: Install assume-role
+        run: go install github.com/remind101/assume-role@latest
+
+      - name: Install Pulumi CLI
+        uses: pulumi/setup-pulumi@v2
```

Comment:
Doesn't really impact the content of this change necessarily, but worth noting that this is now `pulumi/actions@v4` in the workflow on `master`, so we'll want to update this here as well.